NEWS

```diff
@@ -108,6 +108,9 @@
    directories, so that proj can be used as a subdirectory of a larger
    project (#2913)
 
+ o Fix unreachable internal code in Icosahedral Snyder Equal Area projection
+   (#423)
+
```

Comment:
Please leave this bit out. We only update news at release time.

Reply:
Sure thing, sorry about that.  I've also been sending PRs to a different project that requires you to update NEWS yourself, so I just did it out of habit.